README.md

```diff
@@ -72,6 +72,13 @@ You can also install directly from source:
 pip install "id-mava[reverb,jax,envs] @ git+https://github.com/instadeepai/mava.git"
 ```
 
+If you would like to develop features for Mava, the best way to do download all dependencies is:
+
+```bash
+git clone https://github.com/instadeepai/mava.git
+pip install -e "mava[reverb,jax,envs]"
```

Comment:
Maybe don't have all the envs here. Just have `[reverb, jax]` and then add an environment specific example to the detailed install section please?